envisage/ui/single_project/project.py

```diff
@@ -13,9 +13,13 @@
 """
 
 # Standard library imports.
+import datetime
 import logging
+import keyword
```

Comment:
Also unused, I think. (And while we're at it, the existing `import sys` could be removed, too.)